src/components/Hackathon.js

```diff
@@ -1,11 +1,8 @@
-import React, { useEffect } from "react";
+import React from "react";
```

Comment:
Make a new css file "Hackathon.css" in the styles folder like other and put all element styles there instead of using inline css.

Reply:
Take a look at how it's done for other pages like DevChamps.js